### DISCUSSION TITLE
Slow like hell

### DISCUSSION THREAD

**Discussion by keremcozin** [TeamNewPipe/NewPipe#11592] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Wonderful app but downloads are slow like from stone age.

### Additional information

Wonderful app! But it's hellish slow if we want to download something.

**Comment by opusforlife2** [comment]
Try running it on a Ferrari.

**Comment by asandikci** [comment]
@keremcozin 
Try increasing Threads at the bottom of download menu
